src/components/ClickableStyle/ClickableStyle.tsx

```diff
@@ -56,7 +56,8 @@ export type ClickableStyleProps<IComponent extends React.ElementType> = {
     | 'secondary'
     | 'tertiary'
     | 'icon'
-    | 'link'
+    | 'linkDefault'
+    | 'linkStrong'
```

Comment:
the (rare) times I'm seeing multi-word variants in this code base, they've been dash-separated (e.g. `link-default`, `link-strong`). We should use that for consistency unless you feel strongly

Reply:
Ah, I didn't see that; happy to change it 👍